canvas_exercise/shapes_game/index.js

```diff
@@ -23,14 +75,42 @@ window.addEventListener("load", function() {
       expectedKeysMap = {white0: 38, red1: 40, red0: 37, white1: 39},
       timerSpan = document.getElementById("time-remaining"),
       scoreSpan = document.getElementById("score-val"),
-      seconds = 3,
+      seconds = 0,
       intervalId;
 
   canvas.width = width;
   canvas.height = height;
 
-  document.addEventListener("keyup", function() {
- 
+  drawGameStartText(ctx, width, height)
+
+  document.addEventListener("keyup", function(e) {
+    //I feel like this nested if statements can be condensed.
+    if (timerSpan.innerHTML > 29) {
```

Comment:
This should be a better way of tracking this.  If I hit the space bar mutliple times before the time changes to 29, weird things might start happening.  I would just keep a variable to tell if the game has started or not.  I think I called it gameOn in my solution.